kernel/kernel-impl/src/main/sql/oracle/sakai_realm.sql

```diff
@@ -367,7 +367,8 @@ INSERT INTO SAKAI_REALM_FUNCTION VALUES (SAKAI_REALM_FUNCTION_SEQ.NEXTVAL, 'cont
 INSERT INTO SAKAI_REALM_FUNCTION VALUES (SAKAI_REALM_FUNCTION_SEQ.NEXTVAL, 'content.hidden');
 INSERT INTO SAKAI_REALM_FUNCTION VALUES (SAKAI_REALM_FUNCTION_SEQ.NEXTVAL, 'conversations.roletype.instructor');
 INSERT INTO SAKAI_REALM_FUNCTION VALUES (SAKAI_REALM_FUNCTION_SEQ.NEXTVAL, 'conversations.moderate');
-INSERT INTO SAKAI_REALM_FUNCTION VALUES (SAKAI_REALM_FUNCTION_SEQ.NEXTVAL, 'conversations.topic.create');
```

Comment:
this removal correct?

Reply:
> this removal correct?

Yes. You can't see it on the mysql file.
https://github.com/sakaiproject/sakai/blob/master/kernel/kernel-impl/src/main/sql/mysql/sakai_realm.sql#L357